src/Evolution/Systems/CurvedScalarWave/Worldtube/RadiusFunctions.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/RadiusFunctions.hpp"
+
+#include <cmath>
+#include <cstddef>
+
+#include "Utilities/ErrorHandling/Error.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+
+double smooth_broken_power_law(double orbit_radius, double alpha, double amp,
+                               double rb, double delta) {
```

Comment:
[optional] Make these all `const` since you don't change them. Also can remove the extra space above